lib/lintPostcssResult.mjs

```diff
@@ -135,7 +136,7 @@ export default async function lintPostcssResult(stylelintOptions, postcssResult,
  */
 function isFixCompatible({ stylelint }) {
 	// Check for issue #2643
-	if (stylelint.disabledRanges.all && stylelint.disabledRanges.all.length) return false;
+	if (stylelint.disabledRanges[RULE_NAME_ALL]?.length) return false;
 
 	return true;
```

Comment:
```suggestion
	return Boolean(!stylelint.disabledRanges[RULE_NAME_ALL]?.length);
```

Reply:
Make sense. Fixed via ee92462be. I omitted `Boolean()` since it would be redundant.